_posts/2016-04-15-initial-thoughts-after-going-from-angular-to-react.md

```diff
@@ -0,0 +1,93 @@
+---
+layout: post
+title: "Initial Thoughts After Going From angular 1 to react-redux"
```

Comment:
nice forslag. Jeg føler dog at angular-biten er litt interessant å ta med. @gronnbeck what say you?